ops/ccs-ops-misc/load_test/common/data.py

```diff
@@ -80,6 +81,31 @@ def load_mbis():
         configFile = config.load()
         return mbi.loadData()
 
+'''
+Gets the hashed partially adjudicated mbi data and either returns all the data in a list if
+not a distributed test, or takes a percentage of the data to distribute to the current worker
+thread. The percentage of the data in distributed mode depends on the total number of workers
+and the index of the data is dependant on which worker index calls this method.
+'''
+def load_pa_mbis():
+    if setup.is_master_thread():
+        ## Don't bother loading data for the master thread, it doenst run a test
+        return
+    elif setup.is_worker_thread():
+        worker_number = str(os.environ['LOCUST_WORKER_NUM'])
+        num_workers = os.environ['LOCUST_NUM_WORKERS']
+        print(f"Worker {worker_number} starting...")
+        configFile = config.load()
+        full_mbi_list = pambi.loadData()
+        data_per_user = len(full_mbi_list) // int(num_workers)
```

Comment:
How big do we anticipate this list getting?  Will it fit comfortably in memory?

Reply:
The convention being used is it only pulls 10,000, I seemed to have missed that on my query now that I double check though, I'll add that in.